modules/tinymce/CHANGELOG.md

```diff
@@ -34,11 +34,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The content of the dialog body could not be scrolled. #TINY-9668
 - It was possible to remove links in noneditable contents with the 'unlink' editor command. #TINY-9739
 
+## 6.4.2 - TBD
```

Comment:
Was conflict in this file. Added in `## 6.4.2 - TBD` to separate from 6.5 entries